packages/integrations/mdx/test/fixtures/mdx-images/src/pages/index.mdx

```diff
@@ -0,0 +1,3 @@
+![Houston](../assets/houston.webp)
+
```

Comment:
Can we add the following test cases:
- an image with "title"
- test case where we correctly rename the image? Ideally, we should hit the code where we use `replace(/\W/g, '_')`.

Reply:
Will do, thank you for the callout!